### PR TITLE
change mediainfo use ffprobe and fix mediainfo error handle

### PR DESCRIPTION
**Pull request by james58899**
這可能會讓stream檢查失效

### CODE REVIEW

**Comment by mmis1000** [comment]
stream會失效是故意的

**Comment by james58899** [comment]
@mmis1000 我的意思是可能會讓過濾stream失效

**Comment by james58899** [comment]
修好stream過濾了，讓ffprobe讀取音訊長度，如果不存在就拒絕

File: utils/media_info.js [hunk 1]
```diff
@@ -5,22 +5,27 @@ const fixPathname = require('../utils/fix_pathname');
 function parse(url) {
 	url = fixPathname(url);
 	return new Promise(function (resolve, reject) {
-		execFile('ffprobe', ['-v', 'error', '-show_entries', 'stream_tags=title,artist:format_tags=title,artist', url], {timeout: 10000}, function(error, stdout, stderr) {
+		execFile('ffprobe',
```
Comment (licson) [inline utils/media_info.js]: Please fix your formatting here

File: utils/media_info.js [hunk 2]
```diff
@@ -3,21 +3,20 @@ const execFile = require('child_process').execFile;
...
 function parse(url) {
-	url = fixPathname(url);
 	return new Promise(function (resolve, reject) {
 		execFile('ffprobe',
 		['-v', 'error',
 		'-of', 'default=nw=1',
 		'-show_entries', 'stream_tags=title,artist:format_tags=title,artist:format=duration',
-		url],
+		fixPathname(url)],
 		{timeout: 10000},
```
Comment (licson) [inline utils/media_info.js]: Please be aware of spaces between brackets.

File: utils/media_info.js [hunk 3]
```diff
@@ -3,21 +3,20 @@ const execFile = require('child_process').execFile;
 const fixPathname = require('../utils/fix_pathname');
 
 function parse(url) {
-	url = fixPathname(url);
 	return new Promise(function (resolve, reject) {
 		execFile('ffprobe',
```
Comment (licson) [inline utils/media_info.js]: Please be aware of your function arguments and inline array indentations. You can have your reference from `lib/mp3_stream.js`

**Review by licson — APPROVED** [review]

**Pull request merged**